src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -515,6 +517,44 @@ public Task SendFunctionEnvironmentReloadRequest()
             return _reloadTask.Task;
         }
 
+        public Task SendWorkerWarmupRequest()
+        {
+            bool capabilityEnabled = !string.IsNullOrEmpty(_workerCapabilities.GetCapabilityState(RpcWorkerConstants.HandlesWorkerWarmupMessage));
+            if (!capabilityEnabled)
+            {
+                _workerChannelLogger.LogDebug("Worker warmup capability not enabled");
```

Comment:
If worker is not supporting this capability, don't we need to set the result (to false)? Because `ProcessWorkerWarmupResponse` will not execute in that case. Correct? I know we await this method and I am wondering that will handle the tasks status "WaitingForActivation" (The default status) in this case.